src/otx/algo/instance_segmentation/heads/custom_rtmdet_ins_head.py

```diff
@@ -0,0 +1,364 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+"""Custom RTMDetInsSepBNHead for OTX RTMDet-InstSeg instance segmentation models."""
+
+from __future__ import annotations
+
+import math
+from typing import TYPE_CHECKING
+
+import torch
+import torch.nn.functional as F  # noqa: N812
+from mmcv.ops import RoIAlign, batched_nms
+from mmdeploy.codebase.mmdet import get_post_processing_params
+from mmdeploy.codebase.mmdet.models.dense_heads.rtmdet_ins_head import _mask_predict_by_feat_single
+from mmdeploy.core import FUNCTION_REWRITER
+from mmdeploy.mmcv.ops.nms import multiclass_nms
+from mmdet.models.dense_heads.rtmdet_ins_head import RTMDetInsSepBNHead
+from mmdet.registry import MODELS
+from mmdet.structures.bbox import get_box_tensor, get_box_wh, scale_boxes
+from mmengine.config import ConfigDict
+
+if TYPE_CHECKING:
+    from mmengine.structures import InstanceData
+
+
+@MODELS.register_module()
+class CustomRTMDetInsSepBNHead(RTMDetInsSepBNHead):
+    """Custom RTMDet instance segmentation head."""
```

Comment:
Could you add why do we need custom head and what is difference between custom head and origin one?

Reply:
Done. Thanks!